### ISSUE TITLE
Licensing question for iOS deployment

### ISSUE THREAD

**Issue by elix22** [zeromq/netmq#1043]
ZeroMQ is licensed under the same license LGPLv3 but with static linking exception  .
SEE : http://wiki.zeromq.org/area:licensing
Meaning it can be used in closed source commercial iOS applications. (on iOS everything is statically linked)

I didn't find the same license exception in this project .

On IOS all  .NET  assemblies are AOT compiled prior of deployment .
So specifically in this case the netmq assembly is  AOT compiled and statically linked with the iOS binary 
so basically to my understanding it can't be used on IOS for commercial use.

Can someone clarify if this is specific use case 

Thanks

**Comment by elix22** [comment]
Sorry Found the exception in the license itself
Can be closed
